template/en/default/global/header.html.tmpl

```diff
@@ -234,6 +234,9 @@
   <body
         class="[% urlbase.replace('^https?://','').replace('/$','').replace('[-~@:/.]+','-') FILTER css_class_quote %]
                skin-[% user.settings.skin.value FILTER css_class_quote %]
+               [% IF Bugzilla.request_cache.mfa_warning %]
+               mfa-warning
```

Comment:
nit: indent the block, or replace with a one-liner `[% "mfa-warning" IF Bugzilla.request_cache.mfa_warning %]`